build.sbt

```diff
@@ -16,5 +16,6 @@ parallelExecution in Test := false
 
 libraryDependencies ++= Seq(
   "org.scala-lang.modules" %% "scala-java8-compat" % "0.8.0",
-  "com.novocode" % "junit-interface" % "0.11" % "test"
+  "com.novocode" % "junit-interface" % "0.11" % "test",
+  "com.storm-enroute" %% "scalameter" % "0.8.2"
```

Comment:
There are no known issues in using scalameter that I am aware of, and I use it directly in my projects. I am not actively maintaining it, since I am satisfied with the current feature set, which is roughly measuring running time, measuring running time with outlier elimination, measuring running time with GC elimination, measuring memory footprint, measuring method invocation counts, measuring boxing counts, and measuring GC counts.

There is an HTML reporter that outputs a webpage with curves, and there is also a MongoDB reporter that flushes the results into a MongoDB instance.

I personally switched from using the `Bench` frontend to `JBench` frontend which is based on annotations, but both work fine.